dev/io.openliberty.mcp.internal_fat/fat/src/io/openliberty/mcp/internal/fat/protocol/HttpTest.java

```diff
@@ -124,11 +125,38 @@ public void testIncorrectAcceptHeader() throws Exception {
                         }
                         """;
 
-        HttpRequest JsonRequest = new HttpRequest(server, "/httpTest/mcp").requestProp("Accept", "application/json")
-                                                                          .requestProp(MCP_PROTOCOL_VERSION, MCP_PROTOCOL_HEADER)
-                                                                          .jsonBody(request).method("POST").expectCode(406);
+        HttpRequest JsonRequest = new HttpRequest(server, ENDPOINT).requestProp("Accept", "application/json")
+                                                                   .requestProp(MCP_PROTOCOL_VERSION, MCP_PROTOCOL_HEADER)
+                                                                   .jsonBody(request).method("POST").expectCode(406);
 
         String response = JsonRequest.run(String.class);
         assertNull("Expected no response body for 406 Not Acceptable due to incorrect Accept header", response);
     }
+
+    @Test
+    public void testCallWithoutSessionId() throws Exception {
+        String request = """
+                        {
+                          "jsonrpc": "2.0",
+                          "id": 1,
+                          "method": "tools/call",
+                          "params": {
+                            "name": "echo",
+                            "arguments": {
+                              "input": "Hello"
+                            }
+                          }
+                        }
+                        """;
+
+        String response = new HttpRequest(server, ENDPOINT)
+                                                           .requestProp("Accept", ACCEPT_HEADER)
+                                                           .requestProp(MCP_PROTOCOL_HEADER, MCP_PROTOCOL_VERSION)
+                                                           .jsonBody(request)
+                                                           .method("POST")
+                                                           .expectCode(400)
+                                                           .run(String.class);
+
+        assertTrue("Expected 'Missing Mcp-Session-Id' in response body", response.contains("Missing Mcp-Session-Id"));
```

Comment:
```suggestion
        assertThat("Expected 'Missing Mcp-Session-Id' in response body", response, containsString("Missing Mcp-Session-Id"));
```

This will tell us what the response actually was if the test fails.